RELEASING.md

```diff
@@ -59,3 +53,4 @@ Publish the release
   * In our case, ```comgoogleapi```
 * Click the *close*, then *release* button just below the top tabs
 * It will take some time (up to 10 minutes) for the package to transition
+* Run ```./gradlew finalizeRelease```
```

Comment:
There needs to be another section: Updating the versions to the next snapshot versions after the release is done.